### PR TITLE
Added PR Template

### PR DESCRIPTION
**Pull request by manthanand**
# Quality Assurance Checklist 
To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 

## Requirements 
- [x] Followed Coding Style Guide 
- [x] Code Build checks pass 
- [x] No merge conflicts 
- [x] Software feature has test associated with it. 
- [x] Software feature has documentation for it updated.
- [x] Test provides useful information and uses relevant data to accurately represent BPS 
    - NOTE: If test file already exists, use that one 
- [x] 2 Members of the team have already reviewed my PR 
    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
- [x] Did you Have Fun?

## Things to Consider 
- [x] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it. 
+- [ ] Test provides useful information and uses relevant data to accurately represent BPS 
+    - NOTE: If test file already exists, use that one 
+- [ ] 2 Members of the team have already reviewed my PR 
+    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
+
+## Things to Consider 
+- [ ] Even if the above are checked, is this the best way of writing my code? 
+    - It's possible to write code that works, but are there ways to make code more efficient?
```
Comment (tiandahuang) [inline PULL_REQUEST_TEMPLATE.md:18]: add did u have fun

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Requirements 
+- [ ] Followed Coding Style Guide 
+- [ ] Code Build checks pass 
+- [ ] No merge conflicts 
+- [ ] Software feature has test associated with it. 
+- [ ] Test provides useful information and uses relevant data to accurately represent BPS 
+    - NOTE: If test file already exists, use that one 
+- [ ] 2 Members of the team have already reviewed my PR 
+    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first
```
Comment (tiandahuang) [inline PULL_REQUEST_TEMPLATE.md:14]: while we're at it, it's probably time to remove chase from the reviewers list in the readme

**Review by tiandahuang — DISMISSED** [review]

**Review by connorl309 — APPROVED** [review]
all looks good

**Comment by Champers5000** [comment]
Yep looks good

**Pull request merged**